junebug/api.py

```diff
@@ -338,5 +356,70 @@ def create_router(self, request, body):
         ))
 
     @app.route('/health', methods=['GET'])
+    @inlineCallbacks
     def health_status(self, request):
-        return response(request, 'health ok', {})
+        if self.config.rabbitmq_management_interface:
+
+            ids = yield Channel.get_all(self.redis)
```

Comment:
We can probably do away with the `inlineCallbacks` if we chain the `get_queues` function to this one, ie.
```
d = Channel.get_all(self.redis)
d.addCallback(get_queues)
d.addCallback(defer.DeferredList)
d.addCallback(return_queue_results)
return d
```